bolt-design-system/bolt
# Bolt-list missing class on bolt-list-item on web component

## Jira

http://vjira2:8080/browse/BDS-1157

## Summary

Update examples of web components by adding flex example and update web component `bolt-list-item` by adding behavior the same as in twig version.

## Details
Both versions must have the same HTML structure.

Twig version:
![image](https://user-images.githubusercontent.com/16049049/54351137-1d554900-464f-11e9-87ab-19083abf5120.png)

Web component version:
![image](https://user-images.githubusercontent.com/16049049/54351029-df582500-464e-11e9-8a5c-45e8b42863ae.png)

## How to test

Run this code locally and check if the web component version of display flex is the same as twig version by checking rendered HTML.
